src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -814,12 +814,17 @@ internal DefinitionResult GetDefinition(TextDocumentPosition textDocumentPositio
                         bindingTimeout: LanguageService.PeekDefinitionTimeout,
                         bindOperation: (bindingContext, cancelToken) =>
                         {
-                            string schemaName = this.GetSchemaName(scriptParseInfo, textDocumentPosition.Position, scriptFile);
+                            // Handle tokenizing for Peek Definition 
+                            Position tokenPosition = new Position();
```

Comment:
2 suggestions:
- Move the Position creation up to before `Token selectedToken = ScriptDocumentInfo.GetPeekDefinitionToken) and use its line and character values. This minimizes the times we need to convert the line/character counts by adding 1 to them. Every time we do this, we risk an error later since we're tracking in multiple places
- Use the constructor pattern as follow since it's a clean way to create the object :
```
Position tokenPosition = new Position
{
    Line = textDocumentPosition.Position.Line + 1,
    Character = textDocumentPosition.Position.Character + 1
};
```